backend/drone.js

```diff
@@ -0,0 +1,60 @@
+const dgram = require("dgram");
+const wait = require("waait");
+const app = require("express")();
+const http = require("http").Server(app);
+const io = require("socket.io")(http);
+const throttle = require("lodash/throttle");
+const commandDelays = require("./commandDelays");
+
+const PORT = 8889;
+const HOST = "192.168.10.1";
+const drone = dgram.createSocket("udp4");
+drone.bind(PORT);
+
+function parseState(state) {
+  return state.split(":").map((x) => x.split(":"));
+}
+
+const droneState = dgram.createSocket("udp4");
+droneState.bind(8890);
+
+drone.on("message", (message) => {
+  console.log(`--> : ${message}`);
+
+  // io.sockets.emit('status', message.toString());
+});
+
+function handleError(err) {
+  if (err) {
+    console.log("ERROR");
+    console.log(err);
+  }
+}
+
+const commands = ["command", "battery?", "takeoff", "land", "flip l", "down x"];
+
+const i = 0;
```

Comment:
if you're not actively relying on this variable, can you remove it?

Reply:
the same here...